webapp/src/main/webapp/vue-apps/login-internal-onboarding/main.js

```diff
@@ -37,16 +35,16 @@ const urls = [
   `/social/i18n/locale.portal.login?lang=${lang}`
 ];
 
-export function init(params) {
+export function init(id,params) {
```

Comment:
```suggestion
export function init(id, params) {
```

Reply:
done for this file, also applied for other vue components